### PR TITLE
Reload eventStatistics team-wide

### PR DESCRIPTION
**Pull request by matthopson**
Ensure `eventStatistics` are being reloaded team-wide when saving and deleting statistics and statistic data

@glanotte maybe walk through this logic a bit and let me know if anything is overkill?

This returns a LOT more data on these operations, so I want to be sure this is right.

### CODE REVIEW

File: src/persistence.coffee [hunk 1]
```diff
@@ -397,9 +397,22 @@ modifySDK = (sdk) ->
     (statistic, callback) ->
       toRemove = statistic.statisticData.slice()
       toRemove.push statistic.eventStatistics...
+      toRemove.push statistic.memberStatistics...
```
Comment (coreycondardo) [inline src/persistence.coffee]: what dose `...` do?

Comment (matthopson) [inline src/persistence.coffee]: Splat operator (like ruby)

Comment (coreycondardo) [inline src/persistence.coffee]: :+1: thank you

Comment (matthopson) [inline src/persistence.coffee]: @coreycondardo is that a thumbs up to merge or a thumbs up for half-way answering your question?

Comment (coreycondardo) [inline src/persistence.coffee]: so sorry, that was a thank you, not a merge approval.

**Comment by glanotte** [comment]
:+1: from a rubyist

**Comment by amberwebb** [comment]
:+1:

**Pull request merged**